.github/workflows/lint-test.yml

```diff
@@ -0,0 +1,22 @@
+name: lint and test
+
+on: [push]
```

Comment:
I think triggering the actions only inside pull requests and merge events would be nicer. I am not sure how this works but there are these settings https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request